internal/policies/apparmor/apparmor.go

```diff
@@ -1,4 +1,20 @@
 // Package apparmor provides a manager to apply apparmor policies.
+//
+// The policy manager first checks for the existence of the apparmor_parser
+// executable, and proceeds differently if the executable is not found,
+// depending on whether there are confingured entries in the GPO:
+// - no entries: a warning is logged and the manager returns without error
+// - entries: the manager returns an error if apparmor_parser is not found
```

Comment:
I’m unsure we need to precise that, because you say that "If any errors occur during the policy apply process, the manager will attempt…" so if apparmor_parser is not found, this falls under this case. I would just remove this part.

Reply:
yes, but what if the binary is there, but not executable, should we specify it to explicitly or that goes rather to the "if apply fails" dogma? :)

Reply:
We check it's existence with `exec.LookPath()` which explicitly mentions that it looks for an executable file, so even if the binary is there but isn't executable we exit without doing anything.
I agree that we should add this to the doc as well, I'll get on with it.